tests/GenerateSriHtmlTest.php

```diff
@@ -27,15 +23,24 @@ public function it_generates_html_code_with_integrity()
 
     /** @test */
     public function it_generate_html_code_with_credentials_and_integrity()
+    {
+        $hash = hash('sha256', file_get_contents('./tests/files/app.css'), true);
+        $base64Hash = base64_encode($hash);
+
+        $this->assertStringContainsString("integrity='sha256-{$base64Hash}'", Sri::html('files/app.css', true));
+        $this->assertStringContainsString("crossorigin='use-credentials'", Sri::html('files/app.css', true));
+    }
+
+    /** @test */
+    public function it_returns_an_empty_string_in_disabled_environments()
```

Comment:
Can you change the name for something more descriptive of what the test does (return `''` when disabled)?